packages/tempus/src/Tempus_TimeStepControlStrategy.hpp

```diff
@@ -40,16 +42,31 @@ class TimeStepControlStrategy
     Teuchos::RCP<SolutionHistory<Scalar> > /* sh */,
     Status & /* integratorStatus */){}
 
+  /// \name Overridden from Teuchos::Describable
+  //@{
+    std::string description() const override
+    { return "Tempus::TimeStepControlStrategy"; }
+
+    void describe(Teuchos::FancyOStream          &out,
+                  const Teuchos::EVerbosityLevel verbLevel) const override
+    {
+      Teuchos::OSTab ostab(out,2,"describe");
+      out << description() << std::endl;
+    }
+  //@}
```

Comment:
Do we really want to make this a describable? Not much is done.

Reply:
Yeah, I think you are right.  Let me get this with the next issue.  I would like to get this one through.

Reply:
Started a separate branch for the Strategies which removes this.